data-structure/tuple.ipynb

```diff
@@ -143,7 +143,7 @@
     "t.sort(reverse=True) #desceding order sort\n",
     "t = tuple(t)\n",
     "print(t)\n",
-    "type(t)"
+    "type(t)" # it will be print(type(t)) to display its type
```

Comment:
Don't write comments like this.
In interactive mode, it doesn't require print(...) if it's the last statement to be executed.